src/components/data_submission/NIHDataManagement.js

```diff
@@ -0,0 +1,221 @@
+        }),
+        h(FormField, {
+          id: 'consentProcessDidNotAddressFutureUseOrBroadSharing',
+          defaultValue: initialFormData?.alternativeDataSharingPlanReasons?.includes(alternativeDataSharingPlanReasonValues.consentProcessDidNotAddressFutureUseOrBroadSharing),
+          type: FormFieldTypes.CHECKBOX,
+          onChange: onAlternativeDataSharingPlanReasonsChange,
+          toggleText: 'The consent process did not specifically address future use or broad data sharing for samples collected after January 25, 2015',
+        }),
+        h(FormField, {
+          id: 'consentProcessInadequatelyAddressesRisk',
+          defaultValue: initialFormData?.alternativeDataSharingPlanReasons?.includes(alternativeDataSharingPlanReasonValues.consentProcessInadequatelyAddressesRisk),
+          type: FormFieldTypes.CHECKBOX,
+          onChange: onAlternativeDataSharingPlanReasonsChange,
+          toggleText: 'The consent processes inadequately addresses risks related to future use or broad data sharing for samples collected after January 25, 2015',
+        }),
+        h(FormField, {
+          id: 'consentProcessPrecludesFutureUseOrBroadSharing',
+          defaultValue: initialFormData?.alternativeDataSharingPlanReasons?.includes(alternativeDataSharingPlanReasonValues.consentProcessPrecludesFutureUseOrBroadSharing),
+          type: FormFieldTypes.CHECKBOX,
+          onChange: onAlternativeDataSharingPlanReasonsChange,
+          toggleText: 'The consent specifically precludes future use or broad sharing (including a statement that use of data will be limited to the original researchers)',
+        }),
+        h(FormField, {
+          id: 'otherInformedConsentLimitationsOrConcerns',
+          defaultValue: initialFormData?.alternativeDataSharingPlanReasons?.includes(alternativeDataSharingPlanReasonValues.otherInformedConsentLimitationsOrConcerns),
+          type: FormFieldTypes.CHECKBOX,
+          onChange: onAlternativeDataSharingPlanReasonsChange,
+          toggleText: 'Other informed consent limitations or concerns',
+        }),
+        h(FormField, {
+          id: 'otherReasonForRequest',
+          defaultValue: initialFormData?.alternativeDataSharingPlanReasons?.includes(alternativeDataSharingPlanReasonValues.otherReasonForRequest),
+          type: FormFieldTypes.CHECKBOX,
+          onChange: onAlternativeDataSharingPlanReasonsChange,
+          toggleText: 'Other'
+        }),
+      ]),
+      h(FormField, {
+        isRendered: true, // todo: otherReasonForRequest,
+        id: 'alternativeDataSharingPlanExplanation',
+        defaultValue: initialFormData?.alternativeDataSharingPlanExplanation,
+        onChange,
+        title: 'Explanation for request',
+        validators: [FormValidators.REQUIRED]
+      }),
+      h(FormField, {
+        type: FormFieldTypes.FILE,
+        title: 'Upload your alternative sharing plan.',
+        id: 'alternativeDataSharingPlanFile',
+        onChange,
+      }),
```

Comment:
This feature doesn't show the uploaded file name after selecting a file. Can we file a separate ticket for that specific aspect of the `FormField` component?

Reply:
👍 This is how file uploads look on the DAR form mock:
<img width="1065" alt="image" src="https://user-images.githubusercontent.com/17440010/194081185-5bc66e92-5280-495c-b806-1f0eed7e342d.png">
I think this approach makes more sense. I can do this as a separate ticket, or as a part of https://broadworkbench.atlassian.net/browse/DUOS-1755

Reply:
filed ticket!